torch/_dynamo/symbolic_convert.py

```diff
@@ -582,18 +582,25 @@ def wrapper(self: "InstructionTranslatorBase", inst: Instruction):
                 # torch._dynamo.explain() formats this a little nicer, and presents a slightly
                 # more actionable user code pointer
                 if (
-                    graph_break_log.isEnabledFor(logging.DEBUG)
+                    graph_break_log.isEnabledFor(logging.WARNING)
                     and not explain
                     and graph_break_dup_warning_checker.add(frame_loc)
                 ):
                     user_stack_formatted = "".join(traceback.format_list(user_stack))
                     # This log line is exercised from
                     #   python test/dynamo/test_exc.py -k test_graph_break_log
-                    graph_break_log.debug(
-                        "Graph break: from user code at:\n%s",
-                        user_stack_formatted,
-                        exc_info=True,
-                    )
+                    if excp.msg == "Tensor.item":
+                        graph_break_log.warning(
+                            "Graph break: from user code at:\n%s",
+                            user_stack_formatted,
+                            exc_info=True,
```

Comment:
![image](https://github.com/user-attachments/assets/03d99f2d-d616-49db-aa3b-dab5702d0110)

I think you should make this change directly in `tensor.py`. Extract the real stack using `torch._guards.TracingContext.extract_stack()`.